test/connectors/ScratchConnectorSpec.scala

```diff
@@ -53,12 +53,12 @@ class GuidanceConnectorSpec extends BaseSpec {
 
       MockedHttpClient
         .post(endpoint, dummyProcess)
-        .returns(Future.successful(Right(SaveScratchSubmissionResponse(id))))
+        .returns(Future.successful(Right(ScratchResponse(id))))
```

Comment:
You can't see it here, but the test description uses the old name ScratchProcessSubmissionResponse.

Reply:
fixed, will be pushed in next commit